core/blockchain.go

```diff
@@ -1758,6 +1771,7 @@ func (bc *BlockChain) insertChain(chain types.Blocks, verifySeals bool) (int, er
 				"elapsed", common.PrettyDuration(time.Since(start)),
 				"txs", len(block.Transactions()), "gas", block.GasUsed(),
 				"root", block.Root())
+			log.Error("We are not handling Side chains")
```

Comment:
maybe we should remove `SideStatTy` since it seems it's not possible anymore

Reply:
Removed